### PR TITLE
Trigger fixes

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Use new GQL API - pass `serviceName` instead of `subscriber`
- Display ports and paths for connected triggers (in service view) 
- Fix bug that occurred during navigation from service details to services list.

**Related issue(s)**

[Kyma](https://github.com/kyma-project/kyma/pull/9396)
[Bump](https://github.com/kyma-project/kyma/pull/9409)
[Original issue](https://github.com/kyma-project/kyma/issues/9260)

### CODE REVIEW

**Review by akucharska — DISMISSED** [review]

File: core-ui/src/shared/components/EventTriggers/CreateEventTriggerModal.js [hunk 1]
```diff
@@ -31,15 +37,23 @@ export default function CreateEventTriggerModal({
...
-        position="top"
-        trigger="mouseenter"
         tippyProps={{
           distance: 16,
         }}
       >
         {button}
       </Tooltip>
     );
+  } else if (!isLambda && !servicePorts.length) {
```
Comment (magicmatatjahu) [inline core-ui/src/shared/components/EventTriggers/CreateEventTriggerModal.js:47]:
```js
else if (isServiceWithNoPorts)
```

**Review by magicmatatjahu — COMMENTED** [review]
Very good! :)

**Review by magicmatatjahu — DISMISSED** [review]

**Review by magicmatatjahu — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**